.cirrus.yml

```diff
@@ -84,12 +83,12 @@
   only_if: $CIRRUS_USER_COLLABORATOR == 'true' && $CIRRUS_TAG == "" && ($CIRRUS_BRANCH == "master" || $CIRRUS_BRANCH =~ "branch-.*")
   env:
     WS_APIKEY: VAULT[development/kv/data/mend data.apikey]
-  maven_cache:
-    folder: ${CIRRUS_WORKING_DIR}/.m2/repository
+  gradle_cache:
+    folder: ${CIRRUS_WORKING_DIR}/.gradle/caches
   whitesource_script:
     - source cirrus-env QA
-    - source set_maven_build_version $BUILD_NUMBER
-    - mvn clean install -DskipTests
+    - ./gradlew build -x test
```

Comment:
It is good to always add at least `--console plain` in CI for each Gradle execution. Maybe define `GRADLE_OPTS`?